### DISCUSSION TITLE
Different Shadow drop in `IOS` and `Android`.

### DISCUSSION THREAD

**Discussion by premdasvm** [nativewind/nativewind#617] [Q&A]
Hi,

I'm trying to add shadow to a `View` component. initially shadow was working on IOS only, then i read in the issues that i need to pass shadow background color also, so i did that. But when i do that, i'm getting totally different shadow drop in `IOS` and `Android`.

this is how i'm applying the shadows.
```jsx
<View className="p-6 m-3 bg-white border border-gray-200 rounded-3xl shadow shadow-black">
// Contents here
</View>
```
<img width="937" alt="Screenshot 2023-11-06 at 9 13 15 PM" src="https://github.com/marklawlor/nativewind/assets/14838550/883460b5-1d38-46b8-9c8a-5f299931fda4">

**Comment by marklawlor** [comment]
iOS and Android have different shadows, so its impossible to make them identically match - but we try and get them somewhat close.

Can you provide me with the inline styles that you would use to provide a closer match, and I can use that info to adjust our shadow values.

**Reply by premdasvm** [reply]
I've actually utilized the `react-native-shadow-generator` tool [react-native-shadow-generator](https://ethercreative.github.io/react-native-shadow-generator/) to generate shadows, and it delivered the exact results I was aiming for. Additionally, I found 'react-native-shadow-2' at [this GitHub repository](https://github.com/SrBrahma/react-native-shadow-2), and it also provided excellent results.
